cli/wallet/validator.go

```diff
@@ -68,6 +70,7 @@ func newValidatorCommands() []cli.Command {
 				txctx.SysGasFlag,
 				txctx.OutFlag,
 				txctx.ForceFlag,
+				txctx.AwaitFlag,
```

Comment:
We need to actually handle awaiting flags in appropriate NEO-related actions. Currently they are not handled.